astroquery/linelists/cdms/core.py

```diff
@@ -52,6 +53,8 @@ def query_lines_async(self, min_frequency, max_frequency, *,
         molecule : list, string of regex if parse_name_locally=True, optional
             Identifiers of the molecules to search for. If this parameter
             is not provided the search will match any species. Default is 'All'.
+            Note that if the molecule name contains parentheses, they must be
+            escaped.  For exmaple, 'H2C(CN)2' must be specified as 'H2C\\(CN\\)2'.
```

Comment:
hmm, technically this isn't true now... 

also, "exmaple" is when you saw down a tree...